indexer/backup_full_node_ap_northeast_1.tf

```diff
@@ -45,3 +46,8 @@ module "backup_full_node_ap_northeast_1" {
     aws = aws.ap_northeast_1
   }
 }
+
+moved {
+  from = module.backup_full_node_ap_northeast_1
+  to   = module.backup_full_node_ap_northeast_1[0]
+}
```

Comment:
didn't know you can specify this. pretty cool.

when can this block be deleted?

Reply:
We can delete it after all the migration. 
Or we can keep it there as it's recommended by [Hashicorp](https://developer.hashicorp.com/terraform/language/modules/develop/refactoring#removing-moved-blocks)